public/content/en/basics/basic-types.md

```diff
@@ -33,12 +33,12 @@ between floating point types (e.g `double` to `float`)
 is allowed though.
 
 A conversion to another type may be forced by using the
-`cast(TYPE) var` expression. It needs to be used with great care though
+`cast(TYPE) <var>` expression. It needs to be used with great care though
```

Comment:
Var is a keyword in some languages, hence it might be confusing. I am open to all other ideas making things clearer